main/pages/BasePage.py

```diff
@@ -101,3 +104,18 @@ def verify_email_present(self, email):
         from main.common import EmailWrapper
         data = EmailWrapper.get_email(email)
         assert "test" in data
+
+    def move_to_element(self, locator):
+        element = self.wait_element_visible(locator)
+        actions = ActionChains(self.driver)
+        actions.move_to_element(element)
+        # perform the operation on the element
+        actions.perform()
+
+    def logout(self):
```

Comment:
@rohan2794 do we really need to have this in a BasePage?

Reply:
@nrusinko  at some steps where SidePanel Page reference is  not present , there i am calling logout() on BasePage reference